### ISSUE TITLE
The group is not taking actual position on window resize.

### ISSUE THREAD

**Issue by dennisideaonce** [fabricjs/fabric.js#7105]
## Version
4.4.0

## Test Case
https://jsfiddle.net/dennisideaonce/4bqcjfx7/109/

## Information about the environment
Brower, chrome Angular 11

## Steps to reproduce

- Open the above fiddle
- Window resize 

## Expected Behavior

White rectangle(or the drawing board)  and all its objectives needs to be responsive on different resolutions

## Actual Behavior

The group changes its position up to right,

## What I tried

- Grouped all the objects
- calculated % distance from canvas left corner and the group left
- Scaled the group by the ratio
- Reapplied the percentage position (Seems there is some problem in the step)
-  Ungrouped

Basically, I tried all the above steps but still feel there is something missing in my code, any help is highly appreciated.

_On window resize, I have refreshed the page and reloaded the object from the database, I have commented that line of code from the fiddle_

**Comment by dennisideaonce** [comment]
I fixed the issues with set zoom
